integration-tests/tests/singlestore/test-docker.js

```diff
@@ -0,0 +1,113 @@
+import Docker from 'dockerode';
+import mysql2 from 'mysql2/promise';
+import { v4 as uuid } from 'uuid';
+import getPort from 'get-port';
+import retry from 'async-retry';
+import { drizzle } from 'drizzle-orm/singlestore';
+
+const ENABLE_LOGGING = false;
+let db;
+let client;
+
+async function createDockerDB() {
+    const docker = new Docker();
+    const port = await getPort({ port: 3306 });
+    const image = 'ghcr.io/singlestore-labs/singlestoredb-dev:latest';
+
+    const pullStream = await docker.pull(image);
+    await new Promise((resolve, reject) =>
+        docker.modem.followProgress(pullStream, (err) => (err ? reject(err) : resolve()))
+    );
+
+    const singlestoreContainer = await docker.createContainer({
+        Image: image,
+        Env: ['ROOT_PASSWORD=singlestore'],
+        name: `drizzle-integration-tests-${uuid()}`,
+        HostConfig: {
+            AutoRemove: true,
+            PortBindings: {
+                '3306/tcp': [{ HostPort: `${port}` }],
+            },
+        },
+    });
+
+    await singlestoreContainer.start();
+    await new Promise((resolve) => setTimeout(resolve, 5000)); // Delay to ensure container is ready
```

Comment:
nit: I notice this timeout is 5000ms instead of the 4000ms of the other one. Should they be in sync? Same comment applies in both cases though: are we fairly confident that this is enough time here from a potential flakiness perspective?

Reply:
This was a dev script so I removed it, good callout